torchvision/prototype/transforms/functional/_dispatch.py

```diff
@@ -0,0 +1,448 @@
+# THIS FILE IS AUTOGENERATED
+#
+# FROM torchvision/prototype/transforms/functional/dispatch.yaml
+# WITH scripts/regenerate_transforms_dispatch.py
+#
+# DO NOT CHANGE MANUALLY!
+
+from typing import Any, TypeVar, List, Optional, Tuple
+
+import torch
+import torchvision.prototype.transforms.functional as F
+import torchvision.transforms.functional as _F
+from torchvision.prototype import features
+from torchvision.prototype.features import BoundingBoxFormat, ColorSpace
+from torchvision.transforms import InterpolationMode
+
+Dispatcher = F.utils.Dispatcher
+
+# This is just a sentinel to have a default argument for a dispatcher if the feature specific implementations use
+# different defaults. The actual value is never used.
+FEATURE_SPECIFIC_DEFAULT = object()
+
+T = TypeVar("T", bound=features.Feature)
+
+
+__all__ = [
+    "horizontal_flip",
+    "resize",
+    "center_crop",
+    "normalize",
+    "resized_crop",
+    "erase",
+    "mixup",
+    "cutmix",
+    "affine",
+    "rotate",
+    "adjust_brightness",
+    "adjust_saturation",
+    "adjust_contrast",
+    "adjust_sharpness",
+    "posterize",
+    "solarize",
+    "autocontrast",
+    "equalize",
+    "invert",
+]
+
+
+@Dispatcher
+def horizontal_flip(input: T) -> T:
+    """ADDME"""
+    pass
+
+
+@horizontal_flip.implements(features.Image)
+def _horizontal_flip_image(input: features.Image) -> features.Image:
+    output = F.horizontal_flip_image(input)
+
+    return features.Image.new_like(input, output)
+
+
+@horizontal_flip.implements(features.BoundingBox)
+def _horizontal_flip_bounding_box(input: features.BoundingBox) -> features.BoundingBox:
+    intermediate_format = BoundingBoxFormat.XYXY
+    converted_input = F.convert_bounding_box_format(input, old_format=input.format, new_format=intermediate_format)
+
+    output = F.horizontal_flip_bounding_box(converted_input, image_size=input.image_size)
+    output = F.convert_bounding_box_format(output, old_format=intermediate_format, new_format=input.format)
+
+    return features.BoundingBox.new_like(input, output)
+
+
+@Dispatcher
+def resize(
+    input: T,
+    *,
+    size: List[int],
+    interpolation: InterpolationMode = FEATURE_SPECIFIC_DEFAULT,  # type: ignore[assignment]
+    max_size: Optional[int] = None,
+    antialias: Optional[bool] = None,
+) -> T:
+    """ADDME"""
+    pass
+
+
+@resize.implements(features.Image, pil_kernel=_F.resize)
+def _resize_image(
```

Comment:
This is the mid level API. It should never be called directly and thus is not exposed.